pkg/build/webhook/generic/generic_test.go

```diff
@@ -39,7 +39,7 @@ func GivenRequestWithPayload(t *testing.T, filename string) *http.Request {
 }
 
 func GivenRequestWithRefsPayload(t *testing.T) *http.Request {
-	data, err := ioutil.ReadFile("fixtures/post-receive-git.json")
+	data, err := ioutil.ReadFile("../../../../test/fixtures/webhook/generic/post-receive-git.json")
```

Comment:
this is pretty ugly ;-) Can't we have some helper that will hide all these dots here? Something that knows where is the base dir (origin/)